app/views/shipit/merge_status/_merge_queue_button.html.erb

```diff
@@ -9,8 +9,24 @@
 <% else %>
   <%= form_tag enqueue_pull_request_path(stack, pull_request_number), method: 'put', class: classes, data: {remote: true} do %>
     <%= hidden_field_tag 'referrer', params[:referrer] %>
-    <button type="submit" data-disable-with="Adding to merge queue…" class="btn btn-primary">
-      Add to merge queue
-    </button>
+
+    <div class='rollbackability'>
+      <%= label_tag :rollbackable do %>
+        Is this change <%= link_to('safe to rollback?', 'https://development.shopify.io/guides/shipping/safe_to_merge#Signs_your_PR_is_safe_to_rollback', target: :_blank) %>
```

Comment:
Todo: Find a public source or make this link configurable, as the current link is only available to shopify employees I realized.